### DISCUSSION TITLE
it is possible to continue using dynamic imports?

### DISCUSSION THREAD

**Discussion by venturars** [vercel/next.js#42748] [App Router]
it is possible to continue using dynamic imports?

**Comment by rosszurowski** [comment]
Looks like support for `next/dynamic` was [added in the latest 13.1 release](https://nextjs.org/blog/next-13-1#other-improvements):

> `next/dynamic` now uses React primitives `lazy()` and `<Suspense>`. The previous suspense option is no longer required. With these changes, `next/dynamic` is now compatible with the app directory.

I've gotten it working in my own projects as well.
